doc/Language/regexes.pod6

```diff
@@ -1255,6 +1255,47 @@ list of predefined subrules is listed in
 L<S05-regex|https://design.perl6.org/S05.html#Predefined_Subrules> of design
 documents.
 
+=head1 X<Regex Interpolation|regex, Regex Interpolation>
+
+If you want to build a regex using pattern given at the runtime, regex
```

Comment:
Some grammar fixes and this sentence become:

If you want to build a regex using a pattern given at runtime, regex
interpolation is what you are looking for.